app/src/config/feature-flags.ts

```diff
@@ -0,0 +1,143 @@
+import { fetch } from '@forge/api';
+import { Logger } from './logger';
+import envVars, { Environment } from './env';
+
+interface Variation {
+    _id: string;
+    value: boolean;
+}
+
+interface EnvironmentData {
+    on: boolean;
+    archived: boolean;
+    salt: string;
+    sel: string;
+    lastModified: number;
+    version: number;
+    targets: any[];
+    contextTargets: any[];
+    rules: any[];
+    fallthrough: any;
+    offVariation: number;
+    prerequisites: any[];
+    _site: any;
+    _environmentName: string;
+    trackEvents: boolean;
+    trackEventsFallthrough: boolean;
+    _summary: any;
+}
+
+export interface FeatureFlag {
+    name: string;
+    kind: string;
+    description: string;
+    key: string;
+    _version: number;
+    creationDate: number;
+    includeInSnippet: boolean;
+    clientSideAvailability: {
+        usingMobileKey: boolean;
+        usingEnvironmentId: boolean;
+    };
+    variations: Variation[];
+    variationJsonSchema: null | any;
+    temporary: boolean;
+    tags: string[];
+    _links: {
+        parent: {
+            href: string;
+            type: string;
+        };
+        self: {
+            href: string;
+            type: string;
+        };
+    };
+    maintainerId: string;
+    _maintainer: {
+        _links: {
+            self: any;
+        };
+        _id: string;
+        firstName: string;
+        lastName: string;
+        role: string;
+        email: string;
+    };
+    goalIds: string[];
+    experiments: {
+        baselineIdx: number;
+        items: any[];
+    };
+    customProperties: any;
+    archived: boolean;
+    defaults: {
+        onVariation: number;
+        offVariation: number;
+    };
+    environments: {
+        development: EnvironmentData;
+        production: EnvironmentData;
+        staging: EnvironmentData;
+        test: EnvironmentData;
+    };
+}
+
+export default FeatureFlag;
+
+export const LAUNCH_DARKLY_URL = `https://app.launchdarkly.com`;
+const BASE_URL = `${LAUNCH_DARKLY_URL}/api/v2/flags/${envVars.LAUNCHDARKLY_APP_NAME}`;
+
+const baseHeaders = {
+    headers: {
+        Authorization: envVars.LAUNCHDARKLY_API_KEY
+    }
+};
+
+const logger = Logger.getInstance('featureFlags');
+
+async function getFeatureFlag(featureFlagKey: string): Promise<FeatureFlag> {
+    const eventType = 'retrievingFeatureFlag';
+    const errorMsg = `fetching feature flag unexpected status`;
+
+    try {
+        const response = await fetch(`${BASE_URL}/${featureFlagKey}`, { ...baseHeaders });
```

Comment:
Using fetch to call the LD API as we currently can't use the @launchdarkly/node-server-sdk due to Forge's lack of support for the Node.js runtime. https://hello.atlassian.net/wiki/spaces/OTFS/pages/2803689031/Forge+Limitations Work is being done to add this support and we can opt in by being onboarded to a feature flag but doing some removed the ability to run `forge tunnel` locally... Figured this was our next best option.